From 6ebdd1422a0af4ef639e818394af8775fd283838 Mon Sep 17 00:00:00 2001
From: Yann Dirson <yann.dirson@vates.tech>
Date: Wed, 7 Aug 2024 15:32:02 +0200
Subject: [PATCH] openvswitch-config-update: fix python2ism in python3

PR #5261 made the switch to python3 but missed a dict.itervalues() call,
which causes a failure with python3.

Signed-off-by: Yann Dirson <yann.dirson@vates.tech>
---
 scripts/plugins/openvswitch-config-update | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/plugins/openvswitch-config-update b/scripts/plugins/openvswitch-config-update
index d97671b14..834788cf1 100755
--- a/scripts/plugins/openvswitch-config-update
+++ b/scripts/plugins/openvswitch-config-update
@@ -104,7 +104,7 @@ def update(session, args):
     if new_controller:
         query = 'field "management"="true"'
         recs = session.xenapi.PIF.get_all_records_where(query)
-        for rec in recs.itervalues():
+        for rec in recs.values():
             pool_mgmt_macs[rec.get("MAC")] = rec.get("device")
 
     dib_changed = False
-- 
2.39.2

